algorithms/appo/appo.py

```diff
@@ -116,8 +116,11 @@ def add_cli_args(cls, parser):
         p.add_argument('--max_grad_norm', default=4.0, type=float, help='Max L2 norm of the gradient vector')
 
         # components of the loss function
-        p.add_argument('--entropy_loss_coeff', default=0.003, type=float, help='Coefficient for the exploration component of the loss function.')
+        p.add_argument('--exploration_loss_coeff', default=0.003, type=float,
+                       help='Coefficient for the exploration component of the loss function.')
         p.add_argument('--value_loss_coeff', default=0.5, type=float, help='Coefficient for the critic loss')
+        p.add_argument('--exploration_loss', default='entropy', type=str,
+                       help='Choosing the type of exploration loss', choices=['entropy', 'symmetric_kl'])
```

Comment:
I think the longer comment is desirable here. You can use my explanation in the issue or in our Slack conversations :)